server/src/rooms/game/state/index.ts

```diff
@@ -1125,7 +1125,7 @@ export class GameState extends Schema implements GameData {
   
   log(
     message: string,
-    category: string | string = 'Mars Event',
```

Comment:
should we leave the default category value 'Mars Event' in though? I think this might break other `game.log` invocations.

Reply:
fixed all implementations to not utilize default category, can re-add if needed